gsy_framework/schema/avro_schemas/launch_simulation_scenario.json

```diff
@@ -34,6 +34,7 @@
     {"name": "coefficient_percentage", "type": ["null", "float"]},
     {"name": "marketplace_monthly_fee", "type": ["null", "float"]},
     {"name": "allow_external_connection", "type": ["null", "boolean"]},
+    {"name": "is_canary_network", "type": ["null", "boolean"]},
```

Comment:
Late to the party, but this seeks desperately to be changed / refactored from both us and the UI. After all, we can read it directly from the ConfigurationSettings type member

Reply:
I totally agree. Shall I create a ticket?

Reply:
Yes, please do! Let's keep it on the backlog and let's discuss it on our sync with FE next week or on slack, because this requires changes from their side too!

Reply:
https://gridsingularity.atlassian.net/browse/GSYE-563